providers/gong/metadata/schemas.json

```diff
@@ -1,57 +1,64 @@
 {
-  "data": {
-    "calls": {
-      "displayName": "Calls",
-      "fields": {
-        "calendarEventId": "calendarEventId",
-        "clientUniqueId": "clientUniqueId",
-        "customData": "customData",
-        "direction": "direction",
-        "duration": "duration",
-        "id": "id",
-        "isPrivate": "isPrivate",
-        "language": "language",
-        "media": "media",
-        "meetingUrl": "meetingUrl",
-        "primaryUserId": "primaryUserId",
-        "purpose": "purpose",
-        "scheduled": "scheduled",
-        "scope": "scope",
-        "sdrDisposition": "sdrDisposition",
-        "started": "started",
-        "system": "system",
-        "title": "title",
-        "url": "url",
-        "workspaceId": "workspaceId"
-      }
-    },
-    "users": {
-      "displayName": "Users",
-      "fields": {
-        "active": "active",
-        "created": "created",
-        "emailAddress": "emailAddress",
-        "emailAliases": "emailAliases",
-        "extension": "extension",
-        "firstName": "firstName",
-        "id": "id",
-        "lastName": "lastName",
-        "managerId": "managerId",
-        "meetingConsentPageUrl": "meetingConsentPageUrl",
-        "personalMeetingUrls": "personalMeetingUrls",
-        "phoneNumber": "phoneNumber",
-        "settings": "settings",
-        "spokenLanguages": "spokenLanguages",
-        "title": "title",
-        "trustedEmailAddress": "trustedEmailAddress"
-      }
-    },
-    "workspaces": {
-      "displayName": "Workspaces",
-      "fields": {
-        "description": "description",
-        "id": "id",
-        "name": "name"
+  "modules": {
+    "": {
```

Comment:
Not all modules have the same `path`. Rather than that I will do this structure. In case we find out more module properties they will be added to static schema file without many line changes.

Deep connectors with only one module had it named as empty string. I will explicitly set the name to `core`, with constant reference.